bpf/lib/nodeport.h

```diff
@@ -1649,7 +1654,8 @@ static __always_inline int rev_nodeport_lb4(struct __ctx_buff *ctx, int *ifindex
 		}
 #endif
 
-		dmac = map_lookup_elem(&NODEPORT_NEIGH4, &ip4->daddr);
+		if (fib_lookup_bypass())
+			dmac = map_lookup_elem(&NODEPORT_NEIGH4, &ip4->daddr);
```

Comment:
IPv6 counterpart is missing.

Reply:
Missed that woops, added it in just now.